pkgdb2client/admin.py

```diff
@@ -318,34 +344,20 @@ def __handle_request_package(actionid, action):
                     namespace=action['info']['pkg_namespace'],
                 )
 
-        PKGDBCLIENT.handle_api_call(
-            '/admin/action/status',
-            data={
-                'id': actionid,
-                'status': 'Approved'
-            }
-        )
+        approve_action(actionid)
 
-        ns = '%s/' % action['info'].get('pkg_namespace', 'rpms')
+        url = "{0}/package/{1}/{2}".format(
+            PKGDBCLIENT.base_url,
+            action['info'].get('pkg_namespace', 'rpms'),
+            action['info']['pkg_name'])
 
-        url = PKGDBCLIENT.base_url + '/package/' + ns + action['info']['pkg_name']
         utils.comment_on_bug(
             bugid,
             'Package request has been approved: %s' % url
         )
 
     elif decision in ('deny', 'd'):
-        message = input(
-            'Could you explain why you declined this request? (this message '
-            'will be sent to the user)\n=>')
-        data = PKGDBCLIENT.handle_api_call(
-            '/admin/action/status',
-            data={
-                'id': actionid,
-                'status': 'Denied',
-                'message': message,
-            }
-        )
+        data, message = deny_action(actionid)
```

Comment:
at this point both variables are actually used, but I changed it in line 420 (see diff below)